redskyapi/experiments/v1alpha1/http.go

```diff
@@ -426,7 +426,7 @@ func newError(t ErrorType, resp *http.Response, body []byte) error {
 		case http.StatusPaymentRequired:
 			err.Type = ErrUnauthorized
 			if err.Message == "" {
-				err.Message = "account is not activated"
+				err.Message = "your account is not ready, please check https://app.carbonrelay.io/ for more details"
```

Comment:
do we need to flex on the url? ( diff between prod/dev ? )

Reply:
It might be nice to change the `login` command to use `https://staging.redskyops.dev/api/auth_success` and `https://app.carbonrelay.dev/` for non-release builds, but in general the CLI is abstracted from the backend deployment environments.